### PR TITLE
Move to 3.0 Preview 6 SDK

### PR DESCRIPTION
**Pull request by dougbu**
@anurse any reason not to do this?

### CODE REVIEW

File: src/WebHooks/samples/Directory.Build.props [hunk 1]
```diff
@@ -7,5 +7,6 @@
     <StandardTestWebsiteTfms>$(DeveloperBuildTestWebsiteTfms)</StandardTestWebsiteTfms>
     <StandardTestWebsiteTfms Condition=" '$(DeveloperBuild)' != 'true' ">netcoreapp2.1</StandardTestWebsiteTfms>
     <StandardTestWebsiteTfms Condition=" '$(DeveloperBuild)' != 'true' AND '$(OS)' == 'Windows_NT' ">$(StandardTestWebsiteTfms);net461</StandardTestWebsiteTfms>
+    <IsPackable>true</IsPackable>
```
Comment (JunTaoLuo) [inline src/WebHooks/samples/Directory.Build.props]: We don't usually set samples to be packable: https://github.com/aspnet/AspNetCore/blob/4f67ff921673d7367c44ea1840c7a731628bf9bf/Directory.Build.targets#L49

File: global.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "tools": {
-    "dotnet": "3.0.100-preview-010184"
+    "dotnet": "3.0.100-preview6-012264"
```
Comment (JunTaoLuo) [inline global.json]: Why don't we add a requirement for the same SDK version like we do in every other repo?
```json
"sdk": {
    "version": "3.0.100-preview6-012264"
  },
```

**Comment by anurse** [comment]
Always Be Updating ;)

File: src/WebHooks/samples/Directory.Build.props (hunk 1 above)
Comment (dougbu) [reply src/WebHooks/samples/Directory.Build.props]: This is an Web SDK annoyance. It sets `$(WarnOnPackingNonPackableProject)` to `true` for all web projects. I'll try adding `<WarnOnPackingNonPackableProject>false</WarnOnPackingNonPackableProject>` instead.

**Comment by dougbu** [comment]
:up::date: to fix build

**Pull request merged**